elastic/kibana
# [Synthetics] Fixed action connectors and added api test for default alerting

## Summary

Fixes regression introduced in https://github.com/elastic/kibana/pull/160742/files where actions connector were not generated based on rule type 

Fixed action connectors and added e2e test for default alerting

### Testing

- Make sure alerts are being created when a new monitor is added in synthetics 
- Make sure when user adds/remove default connectors in synthetics settings for alert default
- Those connectors are being added/removed in already created rules
- You can view rules within synthetics by clicking on top right Alerts and Rules popover
